Sources/CrowdinSDK/CrowdinSDK/CrowdinSDK.swift

```diff
@@ -236,35 +223,40 @@ extension CrowdinSDK {
     
     /// Method for screenshot feature initialization if Screenshot submodule is added.
     private class func initializeScreenshotFeatureIfNeeded() {
-        if CrowdinSDK.responds(to: Selectors.initializeScreenshotFeature.rawValue) {
-            CrowdinSDK.perform(Selectors.initializeScreenshotFeature.rawValue)
+        let selector =  #selector(CrowdinSDK.initializeScreenshotFeature)
```

Comment:
This method is not available for basic Crowdin SDK functionality. It's a trick to call methods that are defined and implemented in a different part of SDK. Please move that code back if you don't have another idea how to fix that warning. 
Try to use pod 'CrowdinSDK' - you will see all the build errors.

Reply:
This comment related to all changes in this file.

Reply:
I know that the warnings could be annoying, but we do not have ideas for fixing them in a proper way. Ideas appreciated.

Reply:
Have you thought of replacing the code with `void handlers `or `NotificationCenter`?

Reply:
Another resolution could be to move the core needed parts to 'CrowdinSDK'